Engine/TransactionHandlers/BrokerageTransactionHandler.cs

```diff
@@ -322,22 +310,123 @@ public void Exit()
             _exitTriggered = true;
         }
 
+        private Order GetOrderById(int id)
+        {
+            Order order = null;
+
+            if (_orders.TryGetValue(id, out order) == false)
+            {
+                var clientOrder = _algorithm.Transactions.GetOrderById(id);
+
+                if (order != null)
```

Comment:
See my recent change where I moved the storage of the order queue and orders to the transaction handler: https://github.com/QuantConnect/Lean/commit/9c1b0f4616e599f77e33f30c3c1ee8f6b4ef3c74